apps/dockstation/dockstation.yml

```diff
@@ -8,3 +8,4 @@ keywords:
   - Compose
   - Container
 category: 'Developer Tools'
+disabled: true # Dead link(s): https://dockstation.io/
```

Comment:
This still loads for me

Reply:
Oh you're right! It wasn't working for me yesterday in https